.github/workflows/2-find-commit-in-history.yml

```diff
@@ -26,15 +25,9 @@ jobs:
         uses: actions/checkout@v3
       - id: get_step
         run: |
-          echo "::echo::on"
-          echo "::set-output name=current_step::$(cat ./.github/script/STEP)"
-      - id: get_find_commit_id
-        run: |
-          echo "::echo::on"
-          echo "::set-output name=find_commit_id::$(cat ./.github/files/SIDEBARCOMMIT)"
```

Comment:
This command should rather be changed just like the `get_step` command than be deleted.